jupyterhub/jupyterhub/base/jupyterhub-service.yaml

```diff
@@ -14,6 +14,10 @@ spec:
     port: 8081
     protocol: TCP
     targetPort: 8081
+  - name: jsp
```

Comment:
Why do we need this?

Reply:
We were having issues with the api singler-user-profile was using where it was pinging localhost so we switched that to using the jupyterhub service: https://github.com/opendatahub-io/jupyterhub-odh/pull/104/files#diff-15368761312318822f7548285c9c9db73f9b40dc73dcae8192742895d2d8c21eR24

Thats why we had to open up this port